### DISCUSSION TITLE
Mac OS VMs keep rebooting without the ability to install

### DISCUSSION THREAD

**Discussion by xdmx** [quickemu-project/quickemu#1314] [Quickemu]
Hi, I've just discovered Quickemu and installed it on my machine in order to run a Mac OS VM. I've been following the [wiki page](https://github.com/quickemu-project/quickemu/wiki/03-Create-macOS-virtual-machines) and I was able to download the image, but after I select MacOS Base System it starts booting, but then it reboots (or gets stuck like in the screenshot below) and goes back to the boot/selection screen.

I've tried with all versions from Sonoma to Monterrey but all they do the same. I've tried with both 4.9.4 and the current git, and they both do the same. Also after upgrading and rebooting my machine, still nothing. Under both X and Wayland (just in case), and still no change.

Any thoughts on what could be the reason?

```
Quickemu 4.9.5 using /usr/bin/qemu-system-x86_64 v9.0.1
 - Host:     Arch Linux running Linux 6.9.5-arch1-1 host
 - CPU:      AMD Ryzen 7 PRO 4750U with Radeon Graphics 
 - CPU VM:   Haswell-v2, 1 Socket(s), 4 Core(s), 2 Thread(s)
 - RAM VM:   8G RAM
 - BOOT:     EFI (macOS), OVMF (OVMF_CODE.fd), SecureBoot (off).
 - Disk:     macos-sonoma/disk.qcow2 (16G)
             Looks unused, booting from macos-sonoma/RecoveryImage.img
 - Recovery: macos-sonoma/RecoveryImage.img
 - Display:  SDL, VGA, GL (on), VirGL (off) @ (1280 x 800)
 - Sound:    intel-hda (hda-micro)
 - ssh:      On host:  ssh user@localhost -p 22220
 - 9P:       On guest: sudo mount_9p Public-user
 - 9P:       On host:  chmod 777 /home/user/
             Required for macOS integration 👆
 - Network:  User (virtio-net)
 - Monitor:  On host:  nc -U "macos-sonoma/macos-sonoma-monitor.socket"
             or     :  socat -,echo=0,icanon=0 unix-connect:macos-sonoma/macos-sonoma-monitor.socket
 - Serial:   On host:  nc -U "macos-sonoma/macos-sonoma-serial.socket"
             or     :  socat -,echo=0,icanon=0 unix-connect:macos-sonoma/macos-sonoma-serial.socket
 - Process:  Started macos-sonoma.conf as macos-sonoma (97314)

```

```
#!/usr/bin/quickemu --vm
guest_os="macos"
disk_img="macos-sonoma/disk.qcow2"
img="macos-sonoma/RecoveryImage.img"
macos_release="sonoma"
```

![Screenshot_20240625_161632](https://github.com/quickemu-project/quickemu/assets/115298/4207d2d4-6b0b-40ff-b0be-0694a0d6102d)

**Comment by lj3954** [comment]
Issue #1273 is relevant to this.
